subt/zmq-subt-x4.json

```diff
@@ -98,17 +111,23 @@
               ["rosmsg.image", "app.image"],
 
               ["rosmsg.depth", "depth2scan.depth"],
-              ["depth2scan.scan", "app.scan"],
+              ["rosmsg.scan", "app.scan360"],
               ["rosmsg.depth", "detector.depth"],
 
-              ["rosmsg.pose2d", "app.pose2d"],
+              ["rospy.rot", "app.rot"],
```

Comment:
Just a note for a future refactoring - it'd be great if `rot` could be refactored out.

Reply:
+1 for looking this direction in future. pose3d comes with rotation, so no need to also have it separately and to create an opportunity for a mismatch.